rocAL/rocAL/source/augmentations/color_augmentations/node_gamma.cpp

```diff
@@ -42,7 +41,7 @@ void GammaNode::create_node()
         THROW("Uninitialized input/output arguments")
 
     _shift.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
-    _node = vxExtrppNode_GammaCorrectionbatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _shift.default_array(), _batch_size);
+    // _node = vxExtrppNode_GammaCorrection(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _shift.default_array(), _batch_size);
```

Comment:
Why are we having Gamma if we are not calling Tensor support?

Reply:
Removed Gamma